src/repository/mysql.js

```diff
@@ -0,0 +1,22 @@
+import Sequelize from "sequelize";
+
+export default function MysqlORM(dbHost, dbName, dbUser, dbPass, options) {
+  const sequelize = new Sequelize(dbName, dbUser, dbPass, {
+    host: dbHost,
+    operatorsAliases: false,
+    dialect: "mysql",
+    logging: false,
+
+    pool: {
+      max: options.pool.max,
+      min: options.pool.min,
+      acquire: options.pool.acquire,
+      idle: options.pool.idle
+    }
+  });
+
```

Comment:
Since we are using `Sequelize`, what if we also use the define model feature from `Sequelize`? 
so we don't need to create a migration files.

Reply:
Yes, we should, but for boilerplate, we don't know that entities will be created for each app. So it will be the user of this boilerplate that needs to create the migration